Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -300,6 +315,49 @@ public void Download_Error()
 
                 var lastProgress = progressList.LastOrDefault();
                 Assert.That(lastProgress.Status, Is.EqualTo(DownloadStatus.Failed));
+                GoogleApiException exception = (GoogleApiException) lastProgress.Exception;
+                Assert.That(exception.HttpStatusCode, Is.EqualTo(handler.StatusCode));
+                // Just a smattering of checks - if these two pass, it's surely okay.
+                Assert.That(exception.Error.Code, Is.EqualTo(error.Code));
+                Assert.That(exception.Error.Errors[0].Message, Is.EqualTo(error.Errors[0].Message));
+            }
+        }
+
+        [Test]
+        public void Download_Error_PlaintextResponse()
+        {
+            var downloadUri = "http://www.sample.com";
+            var chunkSize = 100;
+            var responseText = "Not Found";
+
+            var handler = new MultipleChunksMessageHandler { ErrorResponse = responseText };
+            handler.StatusCode = HttpStatusCode.NotFound;
+            handler.ChunkSize = chunkSize;
+            // The media downloader adds the parameter...

```

Comment:
So we except an error since we are going to add alt=media twice?
Please elaborate more.